dotnet/src/dotnetframework/GxMail/GxMail.csproj

```diff
@@ -5,11 +5,14 @@
 		<RootNamespace>GeneXus.Mail</RootNamespace>
 		<PackageTags>SMTP Exchange POP3</PackageTags>
 		<PackageId>GeneXus.Mail</PackageId>
+		<UserSecretsId>73342154-596b-45e8-b5d6-a6109df18256</UserSecretsId>
```

Comment:
@claudiamurialdo Could you take a look at this. I am not sure it is coming from... I did not use any kind of certificate at any moment

Reply:
I guess it was automatically added by VS. We should remove it.